pusher-platform-android/src/main/kotlin/com/pusher/platform/network/AndroidConnectivityHelper.kt

```diff
@@ -1,39 +0,0 @@
-package com.pusher.platform.network
-
-import android.content.BroadcastReceiver
-import android.content.Context
-import android.content.IntentFilter
-import android.net.ConnectivityManager
-import com.pusher.platform.android.content.connectivityManager
-import com.pusher.platform.android.content.onReceiveBroadcast
-import java.util.concurrent.LinkedBlockingQueue
-
-class AndroidConnectivityHelper(private val context: Context) : ConnectivityHelper {
```

Comment:
This complicates the library dramatically and adds too many edge cases. What happens if the connection comes back online after the user exits the app? Since we don't attach to the lifecycle and never clear this from the relevant activity, we could receive a broadcast out of lifecycle. It's easier to handle such failures on the client side than trying to be clever here.